scalastyle-config.xml

```diff
@@ -0,0 +1,106 @@
+<scalastyle>
+    <name>Scalastyle standard configuration</name>
+    <check level="warning" class="org.scalastyle.file.FileTabChecker" enabled="true"/>
+    <check level="warning" class="org.scalastyle.file.FileLengthChecker" enabled="true">
+        <parameters>
+            <parameter name="maxFileLength"><![CDATA[800]]></parameter>
+        </parameters>
+    </check>
+    <check level="warning" class="org.scalastyle.file.HeaderMatchesChecker" enabled="true">
+        <parameters>
+            <parameter name="header"><![CDATA[/*
```

Comment:
this is a public repo so this license header for files is different you can see here https://github.com/hmrc/charities-registration-frontend/pull/451/files#diff-fec16ef4db57aefe119007c1bf950c0e633e21e7cc08f04e722dd3d7de4e5814